### ISSUE TITLE
validation error in mzid

### ISSUE THREAD

**Issue by lars20070** [OpenMS/OpenMS#1848]
This `mzid` file [1] leads to the following validation error.
```
Validation error in file 'MB_Uk40.mzid' line 18805 column 30: element 'MzIdentML' declares duplicate identity constraint unique values
Validation error in file 'MB_Uk40.mzid' line 31660 column 30: element 'MzIdentML' declares duplicate identity constraint unique values
Validation error in file 'MB_Uk40.mzid' line 43526 column 30: element 'MzIdentML' declares duplicate identity constraint unique values
```

[1] https://gist.github.com/lars20070/e8cc50ad6286bbdbc897

**Comment by lars20070** [comment]
@mwalzer Can you please comment?

**Comment by lars20070** [comment]
It is based on four fractions i.e. four `mzML`. The tag therefore appears four times.

Guess the fractions are incorrectly written to the `mzid`. But I cannot see what changes are necessary.

**Comment by mwalzer** [comment]
The culprit is ```				<Measure id="Measure_MZ">``` while writing the FragmentationTable

**Comment by lars20070** [comment]
```
<Measure id="Measure_MZ">
```
appears four times in the `mzid` file (once for each fraction) but is expected to be unique. According to  @germa, we simply replace by
```
<Measure id="Measure_MZ_1">
<Measure id="Measure_MZ_2">
<Measure id="Measure_MZ_3">
<Measure id="Measure_MZ_4">
```

**Comment by mwalzer** [comment]
we also should have a test for that, @lars20070  can you please fix a test file for your multiple fractions case? (there might be more than measure id)

**Comment by lars20070** [comment]
Thanks for the fix @mwalzer. I will add the tests.

**Comment by lars20070** [comment]
Resolved by #1849.
